extra/changelog-generator/changelog-generator

```diff
@@ -285,7 +328,11 @@ for repo in repos:
                             commit_msg += match_keyword.group(2)
                 else:
                     log_entry_local = True
-                    log_entry = match_changelog.group(1)
+                    type_match = re.match(
+                        CONVENTIONAL_COMMIT_TYPE_REGEX, commit_msg, re.IGNORECASE
+                    )
+                    cc_prefix = type_match.group(0) if type_match else ""
+                    log_entry = cc_prefix + match_changelog.group(2)
```

Comment:
285 doesn't really have anything to do with `cc_prefix`. 285 makes it possible that a line like `Changelog(fix): fix desription` adds a completely separate entry into the list of entries. This particular snippet makes it so that a `Changelog: all` with a type prefix in the title will be added according to that type.

Doesn`t really matter that cc_prefix is lower case, upper case, or anything in between since we ignore case when extracting the message in `strip_type`.